### PR TITLE
[fluentd-elasticsearch] Allow elasticsearch path to be specified

### PR DESCRIPTION
**Pull request by 0x4c6565**
#### What this PR does / why we need it:
This PR allows the path for elasticsearch to be specified. Currently, only host/scheme/port can be specified

#### Special notes for your reviewer:
None

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://developercertificate.org) signed
- [x] Chart Version bumped (if the pr is an update to an existing chart)
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[fluentd-elasticsearch]`)

### CODE REVIEW

File: charts/fluentd-elasticsearch/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: fluentd-elasticsearch
-version: 4.3.2
+version: 4.3.3
```
Comment (monotek) [inline charts/fluentd-elasticsearch/Chart.yaml]: new feature needs update of minor version (4.4.0)

File: charts/fluentd-elasticsearch/values.yaml [hunk 2]
```diff
@@ -54,6 +54,7 @@ elasticsearch:
   host: "elasticsearch-client"
   logstashPrefix: "logstash"
   port: 9200
+  path: ""
```
Comment (monotek) [inline charts/fluentd-elasticsearch/values.yaml]: have you testet, that the adding of en empty path value does not affect users which are updating from an older chart version?

File: charts/fluentd-elasticsearch/Chart.yaml (hunk 1 above)
Comment (0x4c6565) [reply charts/fluentd-elasticsearch/Chart.yaml]: Bumped minor version

File: charts/fluentd-elasticsearch/values.yaml (hunk 2 above)
Comment (0x4c6565) [reply charts/fluentd-elasticsearch/values.yaml]: The default behavior here is to set the default value to "", which is the default value in the ES plugin. If a user is upgrading from an older chart version, they would use the default path value of ""

**Pull request merged**